testonly/hammer/hammer.go

```diff
@@ -265,6 +265,15 @@ func (s *hammerState) rev(which int) int64 {
 	return s.contents[which].rev
 }
 
+func (s *hammerState) lastRev() int64 {
+	s.mu.RLock()
+	defer s.mu.RUnlock()
+	if s.contents[0].data == nil {
```

Comment:
```
return s.contents[0]
```
I believe go returns the nil element if the map doesn't contain a row for a key.

Reply:
Not sure I follow your suggestion -- `contents` is an array not a map, and `data` being `nil` catches the case where no map contents are yet known.  Or am I misunderstanding?

Reply:
Sorry about the map confusion. I *think* the suggestion would still work though. 

If I'm not mistaken, `s.contents` is an array, not a slice; and `s.contents[0]` always exists. 
Because the uninitialized state of `s.contents[0].rev` is already zero, it shouldn't need it's own check - or is there an odd initialization pattern that would make this assumption not true?

Reply:
Ah, you meant `return s.contents[0].rev` not `return s.contents[0].data`, that makes more sense.  Done.